internal/keepers/util_test.go

```diff
@@ -30,34 +36,35 @@ func TestCryptoShuffler(t *testing.T) {
-		{Name: "Double No Match", Sets: [][]string{{"1", "2", "3"}, {"4", "5", "6"}}, ExpectedResult: []string{"1", "2", "3", "4", "5", "6"}},
-		{Name: "Triple Identical", Sets: [][]string{{"1", "2", "3"}, {"1", "2", "3"}, {"1", "2", "3"}}, ExpectedResult: []string{"1", "2", "3"}},
-		{Name: "Triple No Match", Sets: [][]string{{"1", "2", "3"}, {"4", "5", "6"}, {"7", "8", "9"}}, ExpectedResult: []string{"1", "2", "3", "4", "5", "6", "7", "8", "9"}},
-		{Name: "Double w/ Single Overlap", Sets: [][]string{{"1", "2", "3"}, {"3", "4", "5"}}, ExpectedResult: []string{"1", "2", "3", "4", "5"}},
-		{Name: "Double w/ Single Overlap and Gap", Sets: [][]string{{"1", "2", "3", "6", "7"}, {"3", "4", "5"}}, ExpectedResult: []string{"1", "2", "3", "4", "5", "6", "7"}},
-		{Name: "Double w/ Double Overlap and Gap", Sets: [][]string{{"1", "2", "3", "4", "6", "7"}, {"3", "4", "5"}}, ExpectedResult: []string{"1", "2", "3", "4", "5", "6", "7"}},
-		{Name: "Triple w/ Single Overlap", Sets: [][]string{{"1", "2", "3"}, {"1", "2", "3"}, {"3", "4", "5"}}, ExpectedResult: []string{"1", "2", "3", "4", "5"}},
-		{Name: "Triple w/ Single Overlap and Gap", Sets: [][]string{{"1", "2", "4"}, {"1", "2", "3", "4"}, {"3", "4", "5"}}, ExpectedResult: []string{"1", "2", "3", "4", "5"}},
+		{Name: "Single Matching Set", Sets: [][]testStringer{{"1", "2", "3"}}, ExpectedResult: []testStringer{"1", "2", "3"}},
+		{Name: "Double Identical", Sets: [][]testStringer{{"1", "2", "3"}, {"1", "2", "3"}}, ExpectedResult: []testStringer{"1", "2", "3"}},
+		{Name: "Double No Match", Sets: [][]testStringer{{"1", "2", "3"}, {"4", "5", "6"}}, ExpectedResult: []testStringer{"1", "2", "3", "4", "5", "6"}},
+		{Name: "Triple Identical", Sets: [][]testStringer{{"1", "2", "3"}, {"1", "2", "3"}, {"1", "2", "3"}}, ExpectedResult: []testStringer{"1", "2", "3"}},
+		{Name: "Triple No Match", Sets: [][]testStringer{{"1", "2", "3"}, {"4", "5", "6"}, {"7", "8", "9"}}, ExpectedResult: []testStringer{"1", "2", "3", "4", "5", "6", "7", "8", "9"}},
+		{Name: "Double w/ Single Overlap", Sets: [][]testStringer{{"1", "2", "3"}, {"3", "4", "5"}}, ExpectedResult: []testStringer{"1", "2", "3", "4", "5"}},
+		{Name: "Double w/ Single Overlap and Gap", Sets: [][]testStringer{{"1", "2", "3", "6", "7"}, {"3", "4", "5"}}, ExpectedResult: []testStringer{"1", "2", "3", "4", "5", "6", "7"}},
+		{Name: "Double w/ Double Overlap and Gap", Sets: [][]testStringer{{"1", "2", "3", "4", "6", "7"}, {"3", "4", "5"}}, ExpectedResult: []testStringer{"1", "2", "3", "4", "5", "6", "7"}},
+		{Name: "Triple w/ Single Overlap", Sets: [][]testStringer{{"1", "2", "3"}, {"1", "2", "3"}, {"3", "4", "5"}}, ExpectedResult: []testStringer{"1", "2", "3", "4", "5"}},
+		{Name: "Triple w/ Single Overlap and Gap", Sets: [][]testStringer{{"1", "2", "4"}, {"1", "2", "3", "4"}, {"3", "4", "5"}}, ExpectedResult: []testStringer{"1", "2", "3", "4", "5"}},
```

Comment:
🏄‍♂️ Would there be any value in adding test cases for strings containing letters, or strings composed of only numbers, but prefixed with zeroes? Not necessarily to challenge the changes here, but as an assurance to future developers what the expected outcome should be?

Reply:
deduct accepts any kind of stringer so it just does not care about values. Or do you mean something else?